pipelines/BpsApPipe.yaml

```diff
@@ -0,0 +1,50 @@
+# UPDATE THIS to the path to the pipeline to run
+pipelineYaml: '${AP_PIPE_DIR}/pipelines/ApPipe.yaml'
```

Comment:
Good point. I will move this to a `bps` subdirectory instead of `pipelines` (in line with RFC-775) and add "template" to the name, because it cannot be run as-is.